src/solver/simulation/economy.cpp

```diff
@@ -35,6 +35,8 @@
 #include "simulation.h"
 #include "../optimisation/opt_fonctions.h"
 #include "common-eco-adq.h"
+#include <set>
+#include "../optimisation/adequacy_patch.h"
```

Comment:
In function **Economy::simulationBegin()**, we can see that we create several **PROBLEME_HEBDO**, whereas only one **weeklyOptProblems** is created. 
Indeed, for each year in parallel, we reuse the same **weeklyOptProblems**. 
We should have one **weeklyOptProblems** per **PROBLEME_HEBDO**